sdk/lib/crt/setjmp/arm/setjmp.s

```diff
@@ -55,8 +55,10 @@
     vld1.64 {d6}, [r0]!
     vld1.64 {d7}, [r0]!
 
-    /* Return 1 */
-    mov r0, #1
+    /* Return r1, or 1 if it is 0. */
```

Comment:
This change itself looks good to me,
but `r1` is clobbered (twice) at lines 44-46: `FIXME`.

Reply:
Seems fixed.